src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2329,26 +2330,26 @@ static void window_ride_construction_paint(rct_window* w, rct_drawpixelinfo* dpi
         return;
 
     // Draw track piece
-    x = w->windowPos.x + widget->left + 1;
-    y = w->windowPos.y + widget->top + 1;
+    screenCoords.x = w->windowPos.x + widget->left + 1;
+    screenCoords.y = w->windowPos.y + widget->top + 1;
```

Comment:
Make these assignments a single statement like
```
screenCoords = { , }
```
Delay the variable creation to here
(respecting formatting)